src/Controls/src/Core/HandlerImpl/Label/Label.Windows.cs

```diff
@@ -24,5 +49,56 @@ public partial class Label
 
 		public static void MapMaxLines(ILabelHandler handler, Label label) =>
 			handler.PlatformView?.UpdateMaxLines(label);
+
+		void OnSizeChanged(object sender, UI.Xaml.SizeChangedEventArgs e)
+		{
+			if (Handler is LabelHandler labelHandler)
+			{
+				var platformView = labelHandler.PlatformView;
+				var virtualView = labelHandler.VirtualView as Label;
+
+				if (platformView is null || virtualView is null)
+					return;
+
+				IList<double> inlineHeights = GetInlineHeights();
+				platformView.RecalculateSpanPositions(virtualView, inlineHeights);
+			}
+		}
+
+		IList<double> GetInlineHeights()
+		{
+			IList<double> inlineHeights = new List<double>();
+
+			if (Handler is LabelHandler labelHandler)
+			{
+				var platformView = labelHandler.PlatformView;
+				var virtualView = labelHandler.VirtualView as Label;
+
+				FormattedString formatted = virtualView.FormattedText;
+
+				if (formatted is not null)
+				{
+					var fontManager = virtualView.RequireFontManager();
```

Comment:
virtualView can be null because we used an as casting in line 75. This could throw a NRE.

Reply:
Right, I have applied changes to avoid possible NRE.